Server/mods/deathmatch/logic/luadefs/CLuaDatabaseDefs.cpp

```diff
@@ -20,6 +20,7 @@ void CLuaDatabaseDefs::LoadFunctions()
         {"dbFree", DbFree},
         {"dbPoll", DbPoll},
         {"dbPrepareString", DbPrepareString},
+        {"dbGetConnectionQueueSize", DbGetConnectionQueueSize},
```

Comment:
This is not a review comment, just a note for someone who wants to document this.

**New function**

```lua
int dbGetConnectionQueueSize(element dbConnection)
```

Returns `false` on failure.